portality/api/v1/discovery.py

```diff
@@ -239,59 +230,61 @@ def _make_response(cls, endpoint, res, q, page, page_size, sort,
     def search_articles(cls, q, page, page_size, sort=None):
```

Comment:
All these functions `search_articles` `search_journals` and `search_applications` are now extremely similar.  They pretty much only differ by the config values that they depend on.  So, we can probably refactor them so that there is a single function that does most of the work, and they just pass in the specifics.  That would minimise code duplication.

Reply:
Have replaced with a common search function